### DISCUSSION TITLE
Browser Integration (link not working)

### DISCUSSION THREAD

**Discussion by Ademking** [community/community#9925] [Other Feature Feedback, Questions, & Ideas] [answered]
First of all, I like GitHub Code Search! Keep up the good work. 👍 

The link of "Browser integration" **ProTip** does not work:
![image](https://user-images.githubusercontent.com/12462188/148777019-e0080284-1b57-4732-ab02-88618bd88626.png)

![image](https://user-images.githubusercontent.com/12462188/148777282-ece0fa5d-71c9-4911-a5e1-b1b8533d8339.png)

![image](https://user-images.githubusercontent.com/12462188/148777346-1a6a5b69-853f-4b28-b144-9875bf1eb385.png)

I'm using latest version of Chrome

![image](https://user-images.githubusercontent.com/12462188/148777436-04f604d3-ab0f-4b18-8bbf-51d549f10cfc.png)

**Comment by colinwm** [comment, marked as answer]
Hmm, seems like we're not allowed to link to chrome://settings/searchEngines. I'll write it out then so users can copy/paste into a tab instead.

**Reply by Ademking** [reply]
![image](https://user-images.githubusercontent.com/12462188/148986899-05c1b515-f27f-468d-b2ba-52ffbe512ee3.png)

Thank you @colinwm.
